usage/cli/src/main/license/source-inclusions.yaml

```diff
@@ -20,5 +20,5 @@
 # extras file for org.heneveld.license-audit-maven-plugin
 # listing projects from which *source* files are included
 
-- id: underscore.js
+- id: underscore.js:1.7.0
```

Comment:
our `make-*-license*` scripts combine the relevant `source-inclusions.yaml` files in order to construct the `extrasFile` passed to the plugin.  i've updated `README` to explain this.